src/invoices/InvoicesList/InvoiceLine/InvoiceLineEditLayer.js

```diff
@@ -94,6 +107,8 @@ class InvoiceLineEditLayer extends Component {
               parentMutator={parentMutator}
               onSubmit={this.saveInvoiceLine}
               onCancel={onCloseEdit}
+              vendorCode={vendorCode}
```

Comment:
why it goes as a separate prop? I thought it's an attribute of invoice line

Reply:
It's an attribute of vendor.